api/v1/controllers/samples.js

```diff
@@ -226,10 +234,23 @@ module.exports = {
 
         const j = jobWrapper.createJob(jobType.BULKUPSERTSAMPLES,
           wrappedBulkUpsertData, req);
-      } else if (featureToggles.isFeatureEnabled(constants.featureName)) {
-        redisModelSample.bulkUpsertSample(value);
       } else {
+        const bulkUpsertPromise =
+        featureToggles.isFeatureEnabled(constants.featureName) ?
```

Comment:
There is no change to either of them here. The only change is the use of the ternary operator.